sdk/attestation/azure-security-attestation/src/attestation_administration_client.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include <string>
 
+// cspell: words jwks
```

Comment:
Good candidate for cspell.json override

Reply:
Can I have a per-directory cspell.json file, or is there only the global file? I checked, jwks only appears in the attestation service source code.

Reply:
Resolving my own question: cspell starts at the directory being checked and looks in parent directories for a cspell.json file, and uses that when it finds it. It also appears to use the globally configured cspell.json file.

Reply:
Btw, the fix for this is in a different PR.